tempodb/compactor.go

```diff
@@ -32,6 +32,11 @@ var (
 		Name:      "compaction_errors_total",
 		Help:      "Total number of errors occurring during compaction.",
 	})
+	metricCompactionTracesCombined = promauto.NewCounter(prometheus.CounterOpts{
```

Comment:
let's use "Objects" instead of "Traces" in this metric name and description.  we have an ideal that tempodb is a general purpose kv store built on top of object storage.  unsure how realistic that is :)